presto-product-tests/bin/locations.sh

```diff
@@ -2,3 +2,16 @@ SCRIPT_DIR=${BASH_SOURCE%/*}
 PRODUCT_TESTS_ROOT="${SCRIPT_DIR}/.."
 PROJECT_ROOT="${PRODUCT_TESTS_ROOT}/.."
 DOCKER_CONF_LOCATION="${PRODUCT_TESTS_ROOT}/conf/docker"
+
+# Yes, JDBC_DRIVER_JAR is intentionally set to a directory. 
+# This works better than leaving it empty or unset.
```

Comment:
Also, this should be overridable using an environment variable (without code change). Please use the default value construct, as can be seen in compose-commons.sh